types/column_view.pp

```diff
@@ -0,0 +1 @@
+type Foreman::Column_View = Hash[String,Hash[Enum['title','after','content','title','conditional','eval_content','view','width','custom_method'],String,3,9]]
```

Comment:
What's the 3, 9?

Another way of doing this is just having a ```Foreman::Column_View_Column``` and have the ```Hash[String, Foreman::Column_View_Column]``` type inside the manifest.